app/webpacker/components/Panel/PanelPages.jsx

```diff
@@ -169,7 +169,7 @@ export default {
   },
   [PANEL_PAGES.createNewComers]: {
     name: 'Create New Comers',
```

Comment:
```suggestion
     name: 'Create newcomers',
```
This should be changed as well ;-)

Reply:
Okay now since you said it, I've changed it :-)

As mentioned in https://github.com/thewca/worldcubeassociation.org/pull/10732#discussion_r1932464070, I wanted to change them separately all together in another PR after this is merged. For example, the line above the line you highlighted - for that to be changed, I need to change at many places which will make this PR more confusing and those changes will not be aligned to the PR title.